libr/io/io_bank.c

```diff
@@ -844,6 +865,40 @@ R_API bool r_io_bank_write_at(RIO *io, const ut32 bankid, ut64 addr, const ut8 *
 	return ret;
 }
 
+R_API bool r_io_bank_write_to_overlay_at(RIO *io, const ut32 bankid, ut64 addr, const ut8 *buf, int len) {
+	r_return_val_if_fail (io, false);
+	RIOBank *bank = r_io_bank_get (io, bankid);
+	if (!bank) {
```

Comment:
maybe worth R_LOG_WARN here because if that happens we may miss data without knowing why

Reply:
ok, but it will be a funny warning, we can make it serious when breaking is over

Reply:
yea